### PR TITLE
added mod_app

### PR DESCRIPTION
**Pull request by Brianktoo**
add `mod feature` to find modulo of values

### CODE REVIEW

File: flask-calculator/exponent_app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from flask import Blueprint
+exponential_bp = Blueprint('exponential_bp', __name__)
+from . import routes
```
Comment (codetaskmaster) [inline flask-calculator/exponent_app/__init__.py:3]: Please add whitespace here

File: flask-calculator/exponent_app/exponent.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def exponent(num1,num2):
+    return num1**num2
```
Comment (codetaskmaster) [inline flask-calculator/exponent_app/exponent.py:2]: Here too

File: flask-calculator/exponent_app/__init__.py (hunk 1 above)
Comment (codetaskmaster) [inline flask-calculator/exponent_app/__init__.py:3]: Whitespace please!

File: flask-calculator/exponent_app/exponent.py (hunk 2 above)
Comment (Brianktoo) [reply flask-calculator/exponent_app/exponent.py:2]: done

File: flask-calculator/exponent_app/__init__.py (hunk 1 above)
Comment (Brianktoo) [reply flask-calculator/exponent_app/__init__.py:3]: done

Comment (codetaskmaster) [reply flask-calculator/exponent_app/__init__.py:3]: Check whitespace!

File: flask-calculator/exponent_app/exponent.py (hunk 2 above)
Comment (codetaskmaster) [reply flask-calculator/exponent_app/exponent.py:2]: Not yet please sir. Kindly check again

**Comment by codetaskmaster** [comment]
You have already merged the branch before implementing reviews

**Pull request merged**